### PR TITLE
Remove dealii compatibility fix

### PR DESCRIPTION
**Pull request by EstherHeck**
Pull Request Checklist. Please read and check each box with an X. Delete any part not applicable. Ask on the [forum](https://community.geodynamics.org/c/aspect) if you need help with any step.

*Describe what you did in this PR and why you did it.*

### Before your first pull request:

* [ ] I have read the guidelines in our [CONTRIBUTING.md](../blob/master/CONTRIBUTING.md) document.

### For all pull requests:

* [ ] I have followed the [instructions for indenting my code](../blob/master/CONTRIBUTING.md#making-aspect-better).

### For new features/models or changes of existing features:

* [ ] I have tested my new feature locally to ensure it is correct.
* [ ] I have [created a testcase](http://www.math.clemson.edu/~heister/manual.pdf#sec%3Awriting_tests) for the new feature/benchmark in the [tests/](../blob/master/tests/) directory.
* [ ] I have added a changelog entry in the [doc/modules/changes](../blob/master/doc/modules/changes) directory that will inform other users of my change.

### CODE REVIEW

**Comment by tjhei** [comment]
Hi Esther, can you please change your git username to be "firstname lastname"? See ``git config --global -l``. Find one of the senior developers if you need help with that.

File: source/volume_of_fluid/assembler.cc [hunk 1]
```diff
@@ -112,7 +112,7 @@ namespace aspect
       for (unsigned int face_no = 0; face_no < GeometryInfo<dim>::faces_per_cell; ++face_no)
         {
           // Obtain the normal direction for the face in question
-          // Deall.II orders faces as dim*2+(face_direction_is_positive?1:0)
+          // Deal.II orders faces as dim*2+(face_direction_is_positive?1:0)
```
Comment (tjhei) [inline source/volume_of_fluid/assembler.cc:115]: :-)

File: source/volume_of_fluid/assembler.cc [hunk 2]
```diff
@@ -369,19 +369,12 @@ namespace aspect
...
 
       if ((!face->at_boundary() && !face->has_children())
-          ||
-#if DEAL_II_VERSION_GTE(9,2,0)
-          (face->at_boundary() && neighbor->is_active()))
-#else
-          (face->at_boundary() && neighbor->active()))
-#endif
+          || (face->at_boundary() && neighbor->is_active()))
+
```
Comment (tjhei) [inline source/volume_of_fluid/assembler.cc:373]: can you remove this empty line, please?

File: source/volume_of_fluid/assembler.cc [hunk 3]
```diff
@@ -369,19 +369,12 @@ namespace aspect
...
         {
           if (neighbor->level () == cell->level () &&
-#if DEAL_II_VERSION_GTE(9,2,0)
               neighbor->is_active() &&
-#else
-              neighbor->active() &&
-#endif
+
```
Comment (tjhei) [inline source/volume_of_fluid/assembler.cc:377]: same here, delete the empty line.

**Comment by tjhei** [comment]
The changes look good, but your commits show up as "Author: EstherHeck <hecken@gfz-potsdam.de>". Would you mind changing that? This requires rewriting the history of the commits.

**Comment by EstherHeck** [comment]
in the terminal it now shows my full name. Hope it is ok now?

**Comment by tjhei** [comment]
> Hope it is ok now?

Not quite as your earlier commits were done before the change. Can you grab one of the senior developers to walk you through rewriting the history?

**Comment by EstherHeck** [comment]
will do. we are meeting up with Rene about this anyway before the morning round

**Comment by gassmoeller** [comment]
Looks good. Thank you!

**Pull request merged**